RUNBOOK.md

```diff
@@ -5,6 +5,51 @@
 Production Namespace: aicoe-argocd
 Production Cluster: This information can be found in the internal Runbooks repository.
 
+## Things to look at
+
+We are tracking a number of metrics at our operations dashboard. What follows are the distinct categories of metrics
+
+### Service Level Indicators
+
+SLIs are carefully defined quantitative measures of some aspect of the level of service that is provided. You can find more information on SLIs in the [Site Reliability Engineering Book](https://landing.google.com/sre/sre-book/chapters/service-level-objectives).
+
+| SLI                                    | Query                                                                                                     | Status                        |
```

Comment:
I think it might be useful to have a short description for the non-obvious SLIs in this list

Reply:
I think this still might be necessary. @goern what do you think? You seemed to be particularly interested in understanding the meaning of various metrics that we measure for ArgoCD. Does the documentation in this PR answer your questions?

Reply:
(it's not complete yet) I'm working on updating the documentation for some of the SLIs

Reply:
Added now.